lib/path.js

```diff
@@ -264,6 +264,9 @@ const win32 = {
     const len = path.length;
     if (len === 0)
       return '.';
+    if (len === 1)
+      return path;
```

Comment:
This does not seem right. If `path` is a posix path separator it should return `\\`. That case is also already handled a bit below.